Day 2/DayTwoPartOne.py

```diff
@@ -0,0 +1,43 @@
+import sys
+sys.path.insert(0, "C:/Users/Callipolis/Desktop/Programming/AoC2/Advent-of-Code-2015")
+from InputReader import InputReader
+
+input_reader = InputReader("input2.txt")
+raw_input = input_reader.get_list_with_lines()
+
+class WrapperOrder:
+    
+    paper_counter = 0
+    puzzle_input = None
+    
+    def __init__(self, puzzle_input):
+        self.puzzle_input = puzzle_input
+        self.format_lines()
+
+    def line_to_iterables(self, line):
+        nums = line.split("x")
```

Comment:
Csak hogy kicsit rafossak megint a naming-edre, ez technically nem nums, mert string array. Ha valahol meglatok egy valtozot ami nums akkor az elso tippem egy int array lenne. Legyen akkor num_tokens vagy raw_nums de akkor is inkabb raw_dimensions. 

En ilyen esetben "parts"-nak szoktam hivni (as in the parts of the string that you get after splitting it) vagy tokens-nek (amikor valami string reprezentaciojat tarolom dolgoknak)

Reply:
Ez az első értelmezhető, azonnal tiszta és konkrét naming tanács. :D Köszi, vettem!